opengovsg/ccrm
# Implement integration tests for users routes

- Test CRUD operations for users routes
- Update `getUserById` query to `findOne` instead of `findAll`
- Update `passport`'s `cookieExtractor` to check for undefined cookies

Fixes: #38
